bok_choy/a11y/axe_core_ruleset.py

```diff
@@ -0,0 +1,296 @@
+"""
+Interface for using the axe-core ruleset in A11yAudit.
+See: https://github.com/dequelabs/axe-core
+"""
+import json
+import os
+import requests
+
+from pprint import pformat
+from textwrap import dedent
+
+from .helpers import AccessibilityError
+from ..promise import Promise
+
+
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class AxeCoreAuditConfig(object):
```

Comment:
A cleaner and more extensible implementation would be to think of axe-core and axs as plug-ins and derive them both from an [abstract base class](https://pymotw.com/2/abc/) with abstract methods that are overridden for check_rules, report_errors, etc.